### PR TITLE
Feat/add mobile nav

### PR DESCRIPTION
**Pull request by viru2001**
added mobile navigation

mobile nav (1)

### CODE REVIEW

File: src/frontend/reducer/notesReducer.js [hunk 1]
```diff
@@ -99,6 +100,11 @@ const notesReducer = (notesState, { type, payload }) => {
             ? "NEWEST_FIRST"
             : "",
       };
+    case "MOBILE_SIDEBAR_TOGGLE":
+      return {
+        ...notesState,
+        isMobileSidebarOpen: !notesState.isMobileSidebarOpen,
+      };
```
Comment (vishalg8454) [inline src/frontend/reducer/notesReducer.js:107]: notesReducer.js does not look a good place for `"MOBILE_SIDEBAR_TOGGLE"` to be present. It is related to UI and not to notes.

**Review by vishalg8454 — APPROVED** [review]
Detailed work! Just improve the UI on mobile, sidebar looks big.

File: src/frontend/components/Sidebar/MobileSidebar/MobileSidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+          {pathname === "/notes" && (
+            <fieldset className="mt-4 border-none ml-8 mt-8">
+              <div className="d-flex align-center justify-between">
+                <legend className="headline-sm font-wt-semibold">
+                  Sort By Time
+                </legend>
+                <button
+                  className="btn btn-clear mr-8"
+                  onClick={() => {
+                    notesDispatch({ type: "SORT_BY_TIME", payload: "" });
```
Comment (Shailseen) [inline src/frontend/components/Sidebar/MobileSidebar/MobileSidebar.jsx:90]: need for dispatch empty payload?

**Review by Shailseen — APPROVED** [review]
All looks good to me.

- Given PR description is very useful for reviewer.
- code is understandable looking good

**Review by tanishq20 — APPROVED** [review]
Very well handled with use hook.

File: src/frontend/components/Sidebar/MobileSidebar/MobileSidebar.jsx (hunk 2 above)
Comment (viru2001) [reply src/frontend/components/Sidebar/MobileSidebar/MobileSidebar.jsx:90]: I wanted to make sortBy state an empty string to clear all sorting

File: src/frontend/reducer/notesReducer.js (hunk 1 above)
Comment (viru2001) [reply src/frontend/reducer/notesReducer.js:107]: will think on it

**Pull request merged**